src/components/projects/ManuscriptPageContainer.tsx

```diff
@@ -44,18 +44,22 @@ import { ManuscriptToolbar } from './ManuscriptToolbar'
 import { TrackChangesStyles } from './TrackChangesStyles'
 
 const ManuscriptPageContainer: React.FC = () => {
-  const [{ project, user, permittedActions }] = useStore((state) => {
-    return {
-      project: state.project,
-      user: state.user,
-      permittedActions: state.permittedActions,
+  const [{ project, user, permittedActions, isViewingMode }] = useStore(
+    (state) => {
+      return {
+        project: state.project,
+        user: state.user,
+        permittedActions: state.permittedActions,
+        isViewingMode: state.trackState?.status === 'view-snapshots',
```

Comment:
I think we should just have isViewingMode prop in the store instead of deriving it from trackState status. ops-editor and internal components can then just read it from the store. Potentially there may be other reasons why we might want to enable viewing mode.